README.md

```diff
@@ -152,28 +162,41 @@ with the goal of eventual inclusion into Nix or Nixpkgs.*
 ![RFC Process](./rfcs/0036-rfc-process.png)
 ![Review Process](./rfcs/0036-review-process.png)
 
+### Tips
+
+###### Create Prototypes
+In case your RFC is a technical proposal, you might want to prepare a
+prototype of your idea to firstly make yourself aware of potential pitfalls
+and also help reviewers understand the RFC. Code may be able to explain some
+issues in short.
+
+###### Motivation First
+In any case, it can be a good idea to elaborate the motivation first, while
+leaving the PR in draft. This helps to ensure, that when the solution is
+discussed in the future, interested participants can have a clear and refined
+picture of the problem that you try to address.
```

Comment:
I disagree on this one, as mentioned above.

Also, why is this RFC touching so many parts of the RFC process? It would be better if it only focused on adding the concept of categories, IMO.

Reply:
Some time ago, I acknowledged that two stage discussions are usually more informed and productive.

In my experience, it tremendously helps with forming consensus, when first focusing on the motives and then, after a while, proceeding to the implementation.

If the stages are not separated enough, discussion can (stressfully for all participants) take place on different levels and on different spots on the causality vector. In short: convoluted.

Reply:
I am not opposed to this addition but in the interest of keeping the RFC as small as possible it may be best to defer this or propose it as a separate RFC.